internal/pkg/entities/conversions.go

```diff
@@ -33,33 +32,36 @@ import (
 func ToInfraClusterUpdate(update grpc_public_api_go.UpdateClusterRequest) *grpc_infrastructure_go.UpdateClusterRequest {
 
 	result := &grpc_infrastructure_go.UpdateClusterRequest{
-		OrganizationId: update.OrganizationId,
-		ClusterId:      update.ClusterId,
-		UpdateName:     update.Name != "",
-		Name:           update.Name,
-		AddLabels:      update.AddLabels,
-		RemoveLabels:   update.RemoveLabels,
-		Labels:         update.Labels,
+		OrganizationId:                   update.OrganizationId,
+		ClusterId:                        update.ClusterId,
+		UpdateName:                       update.Name != "",
```

Comment:
```suggestion
		UpdateName:                       update.UpdateName,
```
Empty name may or may not be allowed, but I think we should keep that decision to the request receiver.